pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -74,9 +81,21 @@ let
       rev = "v${version}";
       hash = "sha256-LVLEn+e7c8013pwiLzJiiIObyrlbBHYaioO/SWbItPQ=";
     };
-    });
+  });
+
+  cutlass = fetchFromGitHub {
+    owner = "NVIDIA";
+    repo = "cutlass";
+    rev = "v3.0.0";
+    sha256 = "sha256-YPD5Sy6SvByjIcGtgeGH80TEKg2BtqJWSg46RvnJChY=";
+  };
+
+  # E.g. [ "80" "86" "90" ]
+  cudaArchitectures = (builtins.map cudaFlags.dropDot cudaCapabilities);
+
+  cudaArchitecturesString = strings.concatStringsSep ";" cudaArchitectures;
 in
-stdenv.mkDerivation rec {
```

Comment:
I think it's preferable to use `stdenv` or `backendStdenv` conditionally on `cudaSupport`.

Reply:
Extracted into `buildStdenv` in the let block.